spec/services/reject_application_spec.rb

```diff
@@ -93,5 +93,15 @@
       service.save
       expect(cancel_upcoming_interviews).to have_received(:call!)
     end
+
+    it 'calls the CancelOutstandingReferences service' do
+      cancel_outstanding_references = instance_double(CancelOutstandingReferences, call!: true)
+
+      allow(CancelOutstandingReferences)
+        .to receive(:new).with(application_form: application_choice.application_form)
+                         .and_return(cancel_outstanding_references)
```

Comment:
```
      allow(CancelOutstandingReferences)
        .to receive(:new)
        .with(application_form: application_choice.application_form)
        .and_return(cancel_outstanding_references)
```
por favor

Reply:
rubocop did this identation 🤷 but I am happy to change!

Reply:
Done!

Reply:
Obrigado!